app/common/lib/braveryPanelUtil.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const Immutable = require('immutable')
+
+// Utils
+const urlParse = require('../urlParse')
+
+const getDisplayHost = (lastCommittedURL) => {
+  const parsedUrl = urlParse(lastCommittedURL)
+  if (parsedUrl.protocol === 'https:' || parsedUrl.protocol === 'http:') {
+    return parsedUrl.host
+  }
+
+  return lastCommittedURL
+}
+
+const getRedirectedResources = (redirectedResources) => {
```

Comment:
You have `redirectedResources` in two level array and this function gets second level into one level.

Example:
``` 
{
  google: [
    'google.com',
    'google.si',
  ],
  fb: [
    'facebook.si'
  ]
}
```
into 
```
[
    'google.com',
    'google.si',
    'facebook.si'
]
```